hvac/api/system_backend/key.py

```diff
@@ -320,7 +320,7 @@ def read_backup_keys(self, recovery_key=False):
             url=api_path,
         )
 
-    def cancel_rekey_verify(self):
+    def cancel_rekey_verify(self, recovery_key=False):
```

Comment:
```suggestion
    def cancel_rekey_verify(self, *, recovery_key=False):
```
Let's make all of these keyword-only. The ability to pass a naked boolean value could be unclear, and future-looking, I'd like to try to avoid adding new positional parameters where possible, because it makes adding new parameters or changing ones in the future more strict since re-ordering is a breaking change.

Reply:
Are you willing to trade off the consistency of how we're implementing this in the other methods further up which already have the same parameter. If you're happy for the inconsistency happy to roll it out.

Reply:
Thanks for pointing that out, I didn't see the other methods. I guess we'll leave them positional for consistency. 

If I had a comprehensive plan/guidance written out about this I might've gone the other way, but I haven't been able to write that yet.